src/pages/Loans/LoansOverview/components/BorrowLimit/BorrowLimit.tsx

```diff
@@ -32,9 +32,16 @@ const BorrowLimit = ({
   const { t } = useTranslation();
 
   const { data: currentBorrowLimit, getBorrowLimitUSD } = useAccountBorrowLimit();
+  // ray test touch <<
   const { data: currentLTV, getLTV } = useGetLTV();
```

Comment:
This should be refactored using `LoanCollateralInfo`. Check `useLoanInfo` where you can get `LoanCollateralInfo`.